addons/markers/XEH_preInit.sqf

```diff
@@ -39,7 +39,7 @@ if (isNil QGVAR(MarkerColorsCache)) then {
 
             {
                 if !( _x isEqualType 0) then {
-                    _rgba set [_forEachIndex, call compile _x];
+                    _rgba set [_forEachIndex, parseSimpleArray _x];
```

Comment:
throws script error, `_x` is not an array
might be able to use on _rgba, unless mixed mode arrays are possible?
e.g. `color[] = {1,0,0,"(profilenamespace getvariable ['Map_Unknown_A',0.8])"};`